tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,10 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
+    assert len(jolie.inventory) == 0
+    assert len(fatimah.inventory) == 3
+    assert fatimah.inventory == [item_a, item_b, item_c]
```

Comment:
Again, great work being thorough with your assertions. 

I'd prefer to do 3 separate in checks for each item a, b, c. There was no requirement as to where swapped items end up in the lists, and a test that enforces a particular ordering could be thought of as making too many assumptions about the internal implementation of the swapping function, and could prevent future changes to the implementation.